scout/server/blueprints/phenomodels/controllers.py

```diff
@@ -122,7 +119,7 @@ def _update_subpanel(subpanel_obj, supb_changes):
         all_terms = {}
         # loop over the terms to keep into the checboxes dict
         for child in children_list:
-            if child.startswith("OMIM"):
+            if child.startswith("OMIM" or "ORPHA"):
```

Comment:
```suggestion
            if child.startswith("OMIM") or child.startswith("ORPHA"):
```

You can try also testing locally. The code you wrote causes the error I described as a comment to this PR. Tried with the code in my suggestion and it works!

Reply:
Fixed according to suggestion